### PR TITLE
doc: cluster_tls, wildcard domain correction

### PR DESCRIPTION
**Pull request by raoofm**
doc: cluster_tls, wildcard domain correction

based on the discussion https://github.com/coreos/etcd/issues/8534#issuecomment-329208369

### CODE REVIEW

File: doc/user/cluster_tls.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ The example cluster YAML manifest and example certs can be found in [example/tls
 
 The peer TLS assets should have the following:
 - **peer.crt**: peer communication cert.
-  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc`.
-  In this case, it is `*.example.default.svc`.
+  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc` and `*.{clusterName}.{namespace}.svc.cluster.local`.
```
Comment (hongchaodeng) [inline doc/user/cluster_tls.md]: See https://github.com/coreos/etcd-operator/blob/master/pkg/util/etcdutil/member.go#L40 

After 0.5.1+, etcd operator will not have ".cluster.local" suffix in domain.

**Comment by hongchaodeng** [comment]
I don't understand why you create this PR. Can you elaborate your problem again?

File: doc/user/cluster_tls.md (hunk 1 above)
Comment (raoofm) [reply doc/user/cluster_tls.md]: tls was failing as the host didn't had .cluster.local suffix while creating the cert. I'll try with 0.5.2 and will let you know.

**Comment by hongchaodeng** [comment]
Closing this since it's not necessary.
@raoofm Feel free to reopen if you still have any issue

**Pull request closed without merging**